app/javascript/controllers/complex_checkbox_controller.js

```diff
@@ -0,0 +1,46 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
```

Comment:
Can this be renamed to `date_choice_controller` as it's never used for anything more general?

Reply:
I named this to parallel complex_radio_controller. It is possible that we will want to use in other locations, e.g., to change the authors to use a checkbox instead of radio buttons to be consistent.

Reply:
Can we make that name change then? If that doesn't happen, this premature abstraction makes it harder to read.

Reply:
Done.